xmtp_mls/src/groups/intents.rs

```diff
@@ -363,9 +363,13 @@ impl UpdateGroupMembershipIntentData {
             new_membership.remove(inbox_id)
         }
 
-        new_membership
+        new_membership.failed_installations = new_membership
```

Comment:
Can we just convert this field to a HashSet?

Reply:
let me try, it's everywhere!

Reply:
I have to update protos as well! I would say let's have it for later. wdyt?